samples/gradle.properties

```diff
@@ -4,7 +4,8 @@ kotlin.code.style=official
 org.gradle.parallel=true
 
 # Pin Kotlin version:
-kotlin_version=1.3.30-eap-113
+# CHANGE_VERSION_WITH_RELEASE
+kotlin_version=1.3.30
```

Comment:
Conflicts with https://github.com/JetBrains/kotlin-native/pull/2879

Reply:
We need to use Gradle plugin version `1.3.40-dev-1480`. It has the necessary fixes with arguments, etc.